components/configurator/ui/ConfiguratorContainer.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components'
+
+const ConfiguratorContainer = styled.div`
```

Comment:
Wondering if we should keep using `rem` iinstead of pixels to keep consistency with the rest of the app.

Reply:
Ack, yes definitely - I didn't notice we were doing that, sorry!

We can update in a followup branch, or I can tackle this tomorrow AM and the other stories can branch from here for now.

Reply:
No problem, not a blocker, we can revisit it later. :)